idaes/power_generation/properties/natural_gas_PR.py

```diff
@@ -82,6 +90,18 @@
     },
 }
 
+_phase_dicts_ideal = {
+    "Vap": {
+        "type": VaporPhase,
+        "equation_of_state": Ideal,
+    },
+    "Liq": {
+        "type": LiquidPhase,
+        "equation_of_state": Cubic,
+        "equation_of_state_options": {"type": CubicType.PR}
```

Comment:
I was, but it didn't really work right.  I'll take it out.

Reply:
I didn't really look at it too close.  I just didn't get answers that made sense.  I was trying to separate hydrogen and water.   I asked @dallan-keylogic to look into it, and he probably knows a lot more than I do at this point.

Reply:
@andrewlee94 @eslickj The problem I found with the separation of hydrogen and water was that there was no initialization for flash calculations if there was a single phase component. That's fixed in #555 , but is suspended in limbo now because Andrew wanted tests for Henry's Law initialization, and I didn't have the time to write them last month. I don't have the time to write them now either, but hopefully it will happen sometime soon.